.github/workflows/hosted.yml

```diff
@@ -3,6 +3,8 @@ name: antlr4
 on:
   push:
     branches: [ master, dev, hostedci ]
+  tags:
```

Comment:
The push of a tag will trigger a build which may be not a good solution because this is deviating from the main flow of releases for the other platforms. May be there is a better way to do it.